polling/coinmarketcap.go

```diff
@@ -70,6 +70,9 @@ func (d *CoinMarketCapDataSource) FetchPegPrices() (peg PegAssets, err error) {
 		if !ok {
 			continue
 		}
+		if currency.Symbol != asset && !(asset == "XBT" || asset == "XBC") {
```

Comment:
is there any case where this happens? the only time check is true is if CMC changes a token ID to be a different coin. if that happens, then this error could also occur for xbt/xbc. something like:
```
if currency.Symbol != asset && !(asset == "XBT" && currency.Symbol == "BTC") && !(asset == "XBC" && currency.Symbol == "BCH") {
```

Reply:
This should never happen, but I figured it was a cheap check. I was lazy in checking xbt and xbc, I can include those.

Basically I added this check in another datasouce, then replicated it in a few others as a safety. If we are sure they won't change the token ID I can remove this.